examples/src/main/python/ml/dct_example.py

```diff
@@ -23,6 +23,7 @@
 # $example off$
 from pyspark.sql import SparkSession
 
+
```

Comment:
pep8 recommends 2 empty lines after import statements. `pep8` does it automatically